tests/baselines/reference/constructorOverloads4.errors.txt

```diff
@@ -23,7 +23,7 @@ tests/cases/compiler/constructorOverloads4.ts(11,1): error TS2348: Value of type
     
     (new M.Function("return 5"))();
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2349: Cannot invoke an expression whose type lacks a call signature.
+!!! error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'Function' has no compatible call signatures.

```

Comment:
Specifically, we're not looking for call signatures here. We're looking for a construct signature, so we're not appropriately reporting an error.